exercises/sheet4/marta_bustins_marta_gonzalez_esteve_tarrago/4.tex

```diff
@@ -0,0 +1,186 @@
+
+\documentclass[12pt,a4paper]{article}
+%\input epsf
+\usepackage{wallpaper}
+\usepackage{float}
+\usepackage[T1]{fontenc}
+\usepackage[utf8]{inputenc}
+\usepackage{graphicx}
+\usepackage{amsmath}
+\usepackage[english]{babel}
+\usepackage[small]{caption}
+\usepackage{amsbsy}
+\usepackage{amssymb}
+\usepackage{amsthm}
+\usepackage{amscd}
+\usepackage{amsfonts}
+\usepackage{enumerate}
+\usepackage{subfig}
+\usepackage{ upgreek }
+\usepackage{graphicx}
+\usepackage{listings}
+
+
+
+%\decimalpoint
+\pagestyle{empty}
+\setlength{\voffset}{0cm}
+\setlength{\headheight}{-2.5cm}
+\setlength{\oddsidemargin}{0in}
+\setlength{\textwidth}{6.5in}
+\setlength{\textheight}{9.5in}
+\setlength{\captionmargin}{60pt}
+\setlength{\parindent}{0pt}
+\newcommand{\listaproblemas}[3]{%
+\noindent \rule{\textwidth}{0.02in} \\[0.1cm]
+\noindent {\bf #1} \hfill {\bf #2} \\[0.2cm]
+\centerline{\Large\bf #3}\\
+\noindent \rule{\textwidth}{0.02in} \\[0.1cm]}
+\newcommand{\raya}{\centerline{\rule{0.5\textwidth}{0.02in}}}
+
+%_______________________________________________________________________________
+%                                      Definiciones particulares de este archivo
+
+\usepackage{color}
+\usepackage{colordvi}
+\usepackage{fancyhdr}
+
+\usepackage{paralist}
+\usepackage{url}
+\usepackage{bbm}
+\usepackage{nopageno}
+
+\pagestyle{fancy}
+\fancyhf{}
+\fancyfoot[R]{\vspace{1.75cm} \thepage}
+
+\newtheorem{prob}{}
+\newtheorem*{teo}{Theorem}
+\newtheorem*{defi}{Definition}
+\newtheorem*{obs}{Observation}
```

Comment:
Better to make these non-starred from the outset; the moment you write your second teo/defi/obs, you'll need to do that anyway